include/dos_system.h

```diff
@@ -171,13 +171,24 @@ class DOS_Device : public DOS_File {
 	virtual bool WriteToControlChannel(PhysPt bufptr, uint16_t size,
 	                                   uint16_t* retcode);
 	virtual uint8_t GetStatus(bool input_flag);
+
+	// Indicate or check if the device is managed (or not). Managed devices
+	// are externally owned and managed per RAII. Unmanaged devices will be
+	// destroyed by the device registry on close.
+	void SetManaged();
+	void SetUnmanaged();
+	bool IsUnmanaged() const;
```

Comment:
Nitpicking: `IsManaged`? Just ask for the positive case?

Reply:
I did have `IsManaged()`, but the only use-case (two of them) involves checking  `if(!IsManaged())`, because we both delete *and* null the unmanaged devices, where as we only null the managed devices. 

And I thought `if(IsUnmanaged())` was easier on the eyes, given we have no `IsManaged()` use-cases.

But if we had both use cases - then absolutely, the positive scenario wins.  Will wait for your vote in either direction, now that you can see that there's no usecase for the positive. (I'm happy to switch if you still prefer).

Reply:
> And I thought `if(IsUnmanaged())` was easier on the eyes, given we have no `IsManaged()` use-cases.

Yeah, I thought that's the reason why you did it like this 😄 

I personally prefer to always add the positive check only (`IsActive`, `IsEnabled`, etc.) then I don't need to think about it. Sooner or later we might need an `IsManaged()` check then we need to refactor... which is not the end of the world, of course; I guess I just like to use the "active check" for these types of functions across the board for consistency and "future proofing".

But don't wanna blow it out of proportion, it's a minor thing 😄 Up to you, really.

Reply:
Ah yeah - let's go the positive route 100% of the time; knowing that, it's even easier to read (and have even clearer expectations). Will do!

Reply:
> knowing that, it's even easier to read (and have even clearer expectations)

Yeah, I think so. It's a bit weird to me to read stuff like `!IsActive() && IsUnmanaged()`. Contrived example in this case, but if we mix positive and negative cases, this can sooner or later happen.